<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>218</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>406</width>
    <height>218</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>406</width>
    <height>218</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Radio Player</string>
  </property>
  <property name="windowIcon">
   <iconset resource="buttons.qrc">
    <normaloff>:/buttons/icon.png</normaloff>:/buttons/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(49, 49, 49);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="rathumakaraButton">
         <property name="toolTip">
          <string>RathuMakara FM</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="buttons.qrc">
           <normaloff>:/buttons/rathumakara.png</normaloff>:/buttons/rathumakara.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="kissButton">
         <property name="toolTip">
          <string>Kiss FM</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="buttons.qrc">
           <normaloff>:/buttons/kiss_fm.png</normaloff>:/buttons/kiss_fm.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="nethButton">
         <property name="toolTip">
          <string>Neth FM</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(198, 35, 36);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="buttons.qrc">
           <normaloff>:/buttons/neth.png</normaloff>:/buttons/neth.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="siyathaButton">
         <property name="toolTip">
          <string>Siyatha FM</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(32, 38, 160);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="buttons.qrc">
           <normaloff>:/buttons/siyatha.png</normaloff>:/buttons/siyatha.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="rhythmworldButton">
         <property name="toolTip">
          <string>Rhythm World</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="buttons.qrc">
           <normaloff>:/buttons/rhythm_world.png</normaloff>:/buttons/rhythm_world.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="lakhandaButton">
         <property name="toolTip">
          <string>Lakhanda</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(241, 241, 242);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="buttons.qrc">
           <normaloff>:/buttons/lakhanda.png</normaloff>:/buttons/lakhanda.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="control_btn_wrapper">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="prev_stop_next_wrapper">
         <item>
          <widget class="QToolButton" name="btn_play">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Play</string>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(140, 140, 140);</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="buttons.qrc">
             <normaloff>:/buttons/play.png</normaloff>:/buttons/play.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_stop">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Stop</string>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(140, 140, 140);</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="buttons.qrc">
             <normaloff>:/buttons/stop.png</normaloff>:/buttons/stop.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="playpause_wrapper"/>
       </item>
       <item>
        <widget class="QDial" name="volumedial">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="volumeLabel">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="statuslabel">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Select a Radio Channel to play</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="buttons.qrc"/>
 </resources>
 <connections/>
</ui>
